erpnext/stock/doctype/item/item.py

```diff
@@ -1140,3 +1140,11 @@ def make_purchase_order_item(source_name, target_doc=None):
 	if item.item_defaults and item.item_defaults[0].default_supplier:
 		doc.supplier = item.item_defaults[0].default_supplier
 	return doc
+
+@frappe.whitelist()
+def get_item_price(supplier, item_code):
+	return frappe.db.get_value("Item Price", {
+		"supplier": supplier,
+		"item_code": item_code,
+		"buying": True
```

Comment:
@hrwX  we have item reference too. so its better to add item details in items child table in purchase order.

@neilLasrado this button is in child table of supplier details which add 2 step to create PO. can we bring this button to Item master toolbar let user do multiple PO from dialog box if it has more supplier. @neilLasrado  what do you think?